ods_ci/tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -123,6 +118,9 @@ Verify Oath-Proxy Image Is A CPaaS Built Image
     [Tags]      Sanity
     ...         Tier1
     ...         ODS-666
+    Skip If RHODS Is Self-Managed
+    Wait For Pods To Be Ready    label_selector=deployment=prometheus
+    ...    namespace=${MONITORING_NAMESPACE}    timeout=60s
```

Comment:
this refers to Monitoring NS, but the next lines refers to dashboard pods. Is it expected?

Reply:
You are right. I thought it was the oauth-proxy for prometheus, but as you saw is for dashboard. Fixing it

Reply:
Fixed now